### DISCUSSION TITLE
Can't install "ONScripter-insani" app.

### DISCUSSION THREAD

**Discussion by iopkdj** [insani-org/onscripter-insani#10] [Question/Answer] [answered]
Hello, I downloaded the latest release "onscripter-insani-20230501.macOS.ARM64", when I opened dmg file and double-click the app icon, something got wrong, no any install windows appeared. 
Thank you for your help!!!
 
Model Name:	MacBook Air
  Model Identifier:	MacBookAir10,1
  Model Number:	Z124000CFCH/A
  Chip:	Apple M1
  Total Number of Cores:	8 (4 performance and 4 efficiency)
  Memory:	16 GB
  System Firmware Version:	8422.100.650
  OS Loader Version:	8422.100.650
MacOS: 13.3 (22E252)
https://user-images.githubusercontent.com/60225462/236817435-2ad8a16a-2a4a-49c7-8e52-b7aa499dca2e.mov

**Comment by lightbinder** [comment, marked as answer]
Hi there.  That isn't an installer; it's the app itself.

onscripter-insani is meant to be dragged and dropped into the folder where you already have the following:

- An NScripter script file (```nscript.dat```, ```0.txt```, ```0.utf```, ```00.txt```, ```00.utf```)
- An NScripter archive file (```arc.nsa```), or the actual game assets
- A font file named ```default.ttf``` (onscripter-insani also supports ```.otf```, ```.otc```, and ```.ttc``` font files)

**Reply by iopkdj** [reply]
Thank you !!!
